src/js/utils/form-utils.jsx

```diff
@@ -83,7 +84,7 @@ export const renderFormFields = ({
             :
             <FieldLabel />
         }
-        <div className="col-md-4 col-7 form-element-container">
+        <div className={`form-element-container ${!filterElement ? 'col-md-4 col-7' : 'flex-1 filter-element-container'}`}>
           {renderInput(input, attr)}
```

Comment:
changes which make the position of filter form fields work well without breaking already existing form modals